backup.c

```diff
@@ -185,14 +185,18 @@ do_backup_database(parray *backup_list, pgBackupOption bkupopt)
 	/*
 	 * To take incremental backup, the file list of the latest validated
 	 * full database backup is needed.
+	 * There a problem of increment backup.
```

Comment:
There "is" a ? I think it's better to refer to the issue number.

Reply:
Thank you for you comment. I already fix it..

Reply:
Thanks!